polyfill/index.d.ts

```diff
@@ -516,7 +520,7 @@ export namespace Temporal {
     total(options: DurationTotalOptions): number;
     toLocaleString(locales?: string | string[], options?: Intl.DateTimeFormatOptions): string;
     toJSON(): string;
-    toString(): string;
+    toString(options?: ToStringPrecisionOptions): string;
```

Comment:
Would you like to split this out into a separate PR so that it can be merged immediately?

Reply:
Done. #1500.